python/install_py_bindings.sh

```diff
@@ -2,6 +2,6 @@
 
 set -ex
 
-cargo build -p deltachat_ffi --release
+cargo build -p deltachat_ffi 
```

Comment:
are you sure this should be skipped permanently?

Reply:
yes -- the release wheels are build with full optimization.  and during development it's more important to have fast build cycles.